client/state/ui/guided-tours/test/contexts.js

```diff
@@ -56,4 +58,39 @@ describe( 'selectors', () => {
 			expect( hasUserRegisteredBefore( cutoff )( newUser ) ).to.be.false;
 		} );
 	} );
+
+	describe( '#hasUserPastedFromGoogleDocs', () => {
+		it( 'should return false when no actions', () => {
+			const state = {
+				ui: {
+					actionLog: []
+				}
+			};
+			expect( hasUserPastedFromGoogleDocs( state ) ).to.be.false;
+		});
+
+		it( 'should return false when last action is not the paste event', () => {
+			const state = {
+				ui: {
+					actionLog: [
+						{ type: EDITOR_PASTE_EVENT_FROM_GDOCS},
+						{ type: 'NO_PASTE_EVENT'}
```

Comment:
Missing space before the closing bracket on both lines. Same thing inside the next `it` statement.